eth-custodian/scripts/update_admin_proxy.js

```diff
@@ -0,0 +1,30 @@
+const hre = require('hardhat');
+
+async function updateAdminProxy(ethereumConfig, newAdmin) {
+    const privateKey = hre.network.config.accounts[0];
+    const signerWallet = new hre.ethers.Wallet(privateKey, hre.ethers.provider);
```

Comment:
I think this from the other script is equivalent and an easier way to get the signer
`const [signerWallet] = await hre.ethers.getSigners();`

Reply:
It returns HardhatEthersSigner, but I need Wallet.

Reply:
Signer seem to be enough for all subsequent uses of this object